index.html

```diff
@@ -1799,13 +1799,13 @@ <h2>Trust Model</h2>
           </li>
           <li>
 The <a>subject</a> and <a>verifier</a> trust the <a>issuer</a> to issue
-true (that is, not false) <a>credentials</a> about the <a>subject</a>, and to
-revoke them quickly when appropriate.
+true (that is, not false) <a>verifiable credential</a> about the <a>subject</a>,
+and to revoke them quickly when appropriate.
           </li>
           <li>
-The <a>holder</a> trusts the <a>repository</a> to store the <a>credentials</a>
-securely, to not release them to anyone other than the <a>holder</a>, and to
-not corrupt or lose them while they are in its care.
+The <a>holder</a> trusts the <a>repository</a> to store the
+<a>verifiable credential</a> securely, to not release them to anyone other than
```

Comment:
"verifiable credential" should be "verifiable credentials".

Reply:
Fixed.